languages/julia/exercises/concept/vehicle-purchase/.docs/introduction.md

```diff
@@ -0,0 +1,149 @@
+There are three primary conditional statements that are used in Julia:
+- `if`-statements
+- The ternary operator
+- Short-circuit evaluation
+
+## `if`-statements
+
+`if`-statements in Julia are similar to those seen in other languages:
+
+```julia
+julia> function ispositive(n)
+           if n > 0
+               println("n is positive!")
+           else
+               println("n is not positive!")
+           end
+       end
+ispositive (generic function with 1 method)
+
+julia> ispositive(10)
+n is positive!
+
+julia> ispositive(-10)
+n is not positive!
+```
+
+<!-- TODO: Add that fancy concept highlight embed thing to boolean expression -->
+If the boolean expression following the `if` evalutes to `true`, the first block of code is run and the second block is skipped.
+If the boolean expression following the `if` evalutes to `false`, the first block of code is skipped and the second block is run.
+The program continues running at the first line of code after the `end` keyword.
+
+!!! info
+    In Julia, the `end` keyword signifies the end of all block expressions.
+    This syntax is not specific to `if`-statements or function definitions.
+
+In cases where the second block of code would be just another `if`-statement, `elseif` allows us to avoid nesting `if`-statements within the block:
+
+```julia
+julia> function dessert(fruit)
+           if fruit == "apple"
+               return "Apple Crumble"
+           elseif fruit == "lemon"
+               return "Lemon Meringue Pie"
+           else
+               return "Fruit Salad"
+           end
+       end
+dessert (generic function with 1 method)
+
+julia> dessert("apple")
+"Apple Crumble"
+
+julia> dessert("lemon")
+"Lemon Meringue Pie"
+
+julia> dessert("peach")
+"Fruit Salad"
+```
+
+If an `if`-statement only needs to perform code for one of the cases, there's no need to write out the `else` branch.
+
+## Ternary operator
+
+Simple `if-else` statements can also be written using the ternary operator `a ? b : c`.
+If `a` is true, then `b` will be evaluated.
+Otherwise evaluate `c`.
+
+For example the following function
+
+```julia
+julia> function diagnose(heartrate)
+           if heartrate > 100
+               println("The patient has an elevated heart rate.")
+           else
+               println("The heart rate is nominal or the patient is dead.")
+           end
+       end
+diagnose (generic function with 1 method)
+```
+
+can also be written as
+
+```julia
+julia> diagnose(heartrate) = heartrate > 100 ? println("The patient has an elevated heart rate.") : println("The heart rate is nominal or the patient is dead.")diagnose (generic function with 1 method)
```

Comment:
This is quite a long line, which makes reading it possibly a bit anooying with the horizontal scroll bar. You could consider shortening the text a bit, e.g.:

```suggestion
julia> diagnose(heartrate) = heartrate > 100 ? println("Elevated heart rate") : println("Nominal heart rate or dead")diagnose (generic function with 1 method)
```